xml/docu_styleguide.smartdocs.xml

```diff
@@ -0,0 +1,353 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<sect1 xmlns="http://docbook.org/ns/docbook" 
+       xmlns:xi="http://www.w3.org/2001/XInclude" 
+       xmlns:xlink="http://www.w3.org/1999/xlink" 
+       xmlns:its="http://www.w3.org/2005/11/its" version="5.0" xml:id="sec-smartdocs">
+
+ <title>Working with Smart Docs</title>
```

Comment:
Just an idea: as the term "Smart Docs" is used quite often, define and use an entity for this?

Reply:
Good idea, I'll follow it!